.env

```diff
@@ -10,7 +10,7 @@
 #MySQL 5.6, 5.7, 8.0, Maria 10.0
 # DATABASE_URL="mysql://root:prisma@localhost:3309/prisma"
 # DATABASE_URL="mysql://root:prisma@localhost:3306/prisma"
-DATABASE_URL="mysql://root:prisma@localhost:3307/prisma"
+DATABASE_URL="mysql://root:prisma@localhost:3306/prisma"
```

Comment:
Generally speaking, I'd rather this be left as is, i.e. uncommenting / commenting is fine rather than manually editing ports